src/pages/index.js

```diff
@@ -59,11 +59,11 @@ api
     console.error(err);
   });
```

Comment:
Needs correction: please remove the initial cards request on line 43, since you receive all the data on line 62

Reply:
I removed the initial card request from line 43.